### ISSUE TITLE
Typo DCGAN Notebook

### ISSUE THREAD

**Issue by sridhar912** [udacity/dlnd-issue-reports#292]
There is a typo in the function "discriminator(x, reuse=False, alpha=0.2):"

x1 = tf.layers.conv2d(x, 64, 5, strides=2, padding='same')
relu1 = tf.maximum(alpha * x1, x1)
# 16x16x32  --> This should be 64 instead of 32
